### PR TITLE
Girlcode page - added more content and updated styling for small blocks

### PR DESCRIPTION
**Pull request by afkatja**
- added blocks for more events
- updated contact info
- adjusted styling of the small blocks, also on mobile

### CODE REVIEW

File: styles/style.styl [hunk 1]
```diff
@@ -341,7 +341,7 @@ a
   max-width 33.3333%
 
 .block-small
-  flex 1
+  flex 0 0 320px
```
Comment (Primigenus) [inline styles/style.styl]: This doesn't appear to work as intended, can you remove it? Also, please limit a PR about content to just the content.

Comment (afkatja) [inline styles/style.styl]: is the intention that the block, if alone, will grow to fill the whole container?

File: views/en/page/girlcode.html [hunk 2]
```diff
@@ -43,15 +43,15 @@ <h1 style="display: none;">Girl Code at Q42</h1>
...
             bias in tech, <a href="https://twitter.com/afkatja">Katja</a> (developer)
             asked <a href="https://twitter.com/FYIneke">Ineke</a> (PR): "Can't we
             make this bigger than just one talk for Q'ers only?". Ineke answered,
             "We can and we should!". And so they did. Girl Code was born and the
             first meetup was planned.</p>
-          <p>And in our own world of code? Since last year, the percentage of women
-            at Q42 has risen from just 6% to 15%, and we're not done! So, check out
-            our <a href="//www.q42.nl/vacatures">vacancies</a> (Dutch) and send us your code!</p>
+          <p>And in our own world of code? The percentage of programming women
+            at Q42 is still way below <a href="http://www.mailswitch.nl/files/1638-b42440b5fca651165e886d684da2520c.pdf">the average 10% in the Netherlands</a> (Dutch), so we're not there yet. Do you want to improve these statistics, check out
```
Comment (Primigenus) [inline views/en/page/girlcode.html]: Perhaps explain what the link is for? Right now it just drops you in a PDF with no further explanation.

File: views/en/page/girlcode.html [hunk 3]
```diff
@@ -81,8 +81,8 @@ <h1 style="display: none;">Girl Code at Q42</h1>
           </div>
           <div class="block-small">
             <div class="visual caption">
-              <img src="{{assetsUrl}}/images/girlcode/girlcodemeetsmeteor.png" alt="">
-              <p>"It works!!!"</p>
+              <img src="{{assetsUrl}}/images/girlcode/girlcodemeetsmeteor.png" alt="GirlCode meets Meteor rerun">
```
Comment (Primigenus) [inline views/en/page/girlcode.html]: Illustrative images need no alt attribute value.

File: views/en/page/girlcode.html [hunk 4]
```diff
@@ -50,11 +50,11 @@ <h1 style="display: none;">Girl Code at Q42</h1>
             "We can and we should!". And so they did. Girl Code was born and the
             first meetup was planned.</p>
           <p>And in our own world of code? The percentage of programming women
-            at Q42 is still way below <a href="http://www.mailswitch.nl/files/1638-b42440b5fca651165e886d684da2520c.pdf">the average 10% in the Netherlands</a> (Dutch), so we're not there yet. Do you want to improve these statistics, check out
+            at Q42 is still way below the average 10% in the Netherlands (based on <a href="http://www.mailswitch.nl/files/1638-b42440b5fca651165e886d684da2520c.pdf" target="_blank">the data</a> published by the Dutch employment bureau in 2015  (Dutch), so we're not there yet. Do you want to improve these statistics, check out
             our <a href="//q42.nl/vacatures">vacancies</a> (Dutch) and send us your code!</p>
         </div>
           <div class="visual caption">
-            <img src="{{assetsUrl}}/images/girlcode/ineke-katja.jpg" alt="">
+            <img src="{{assetsUrl}}/images/girlcode/ineke-katja.jpg">
```
Comment (Primigenus) [inline views/en/page/girlcode.html]: They need no attribute _value_. They do need an alt attribute, but it should be empty. You don't need to revert stuff that was already correct on the page :)

**Pull request merged**